packages/nextjs/pages/extensions.tsx

```diff
@@ -107,34 +103,55 @@ export const getStaticProps: GetStaticProps<ExtensionsListProps> = async () => {
 
     const response = await fetch(`${BGAPP_API_URL}/builds?type=extension`);
     const data = await response.json();
-    const formattedExtensions = data.map((ext: any) => {
+    const thirdPartyExtensions = data.map((ext: any) => {
       const githubUrlParts = ext.branch.split("/");
       const githubUsername = githubUrlParts[3];
       const repoName = githubUrlParts[4];
 
+      let installCommand = `npx create-eth@latest -e ${githubUsername}/${repoName}`;
+
+      if (githubUrlParts.length > 5) {
+        const branch = githubUrlParts[6];
+        installCommand = `${installCommand}:${branch}`;
+      }
+
       return {
         name: ext.name,
         description: ext.desc,
         github: ext.branch,
-        installCommand: `npx create-eth@latest -e ${githubUsername}/${repoName}`,
+        installCommand,
         builder: ext.builder,
         coBuilders: ext.coBuilders || [],
         youtube: ext.videoUrl || null,
       };
     });
 
+    const responseCuratedExtensions = await fetch(
+      "https://raw.githubusercontent.com/scaffold-eth/create-eth/refs/heads/main/src/extensions.json",
+    );
+    const dataCuratedExtensions = await responseCuratedExtensions.json();
+
+    const curatedExtensions = dataCuratedExtensions.map((ext: any) => {
+      if (!ext.name) {
+        ext.name = ext.branch;
+      }
+      return ext;
```

Comment:
Just realized github link is not showing for curated extensions. Maybe tweak this?
```
      }
      ext.github = `${ext.repository}/tree/${ext.branch}`;
      return ext;
```

Reply:
Nice catch!! 

Fixed, with something similar since the branch is optional.